Explorer/Assets/DCL/Passport/Modules/UserBasicInfo_PassportModuleController.cs

```diff
@@ -69,11 +69,13 @@ public void Setup(Profile profile)
             //CheckForEditionAvailabilityAsync(checkEditionAvailabilityCts.Token).Forget();
         }
 
-        public void Clear() { }
+        public void Clear()
+        {
+            checkEditionAvailabilityCts.SafeCancelAndDispose();
+        }
```

Comment:
In the clear you can probably put also 
```
view.CopyNameWarningNotification.Hide(true);
view.CopyWalletWarningNotification.Hide(true);
```
Because if you close the passport while they are on they might stay visible once you re open it

Reply:
Totally true! Done here: https://github.com/decentraland/unity-explorer/pull/1684/commits/76bea0c17f7c90c9eb24146faca84458d13cd145